examples/Inference in Bayesian Networks.ipynb

```diff
@@ -17,27 +17,18 @@
     {
      "name": "stdout",
      "output_type": "stream",
-     "text": [
-      "--2016-09-30 12:38:47--  http://www.bnlearn.com/bnrepository/asia/asia.bif.gz\r\n",
-      "Resolving www.bnlearn.com (www.bnlearn.com)... 176.58.124.98\r\n",
-      "Connecting to www.bnlearn.com (www.bnlearn.com)|176.58.124.98|:80... connected.\r\n",
-      "HTTP request sent, awaiting response... 200 OK\r\n",
-      "Length: 310 [application/octet-stream]\r\n",
-      "Saving to: ‘asia.bif.gz’\r\n",
-      "\r\n",
-      "\r",
-      "asia.bif.gz           0%[                    ]       0  --.-KB/s               \r",
-      "asia.bif.gz         100%[===================>]     310  --.-KB/s    in 0s      \r\n",
-      "\r\n",
-      "2016-09-30 12:38:47 (16,4 MB/s) - ‘asia.bif.gz’ saved [310/310]\r\n",
-      "\r\n"
-     ]
     }
    ],
    "source": [
     "# Fetching the network\n",
-    "!wget http://www.bnlearn.com/bnrepository/asia/asia.bif.gz\n",
-    "!gzip -qd asia.bif.gz | rm asia.bif.gz"
+    "f=wget.download('http://www.bnlearn.com/bnrepository/asia/asia.bif.gz')",
+    
+    "f = gzip.open('asia.bif.gz', mode='rb')"
```

Comment:
You can use the `with` statement to open the file so that you don't need to close the file explicitly. Will make the code cleaner.